style.css

```diff
@@ -0,0 +1,476 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+html {
+  font-size: 15px;
+  font-family: 'montserratregular', sans-serif;
+}
+
+body {
+  margin: 0 auto;
+  position: relative;
+  min-width: 320px;
+  max-width: 1440px;
+}
+
+h1,
+h2 {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  font-family: 'montserratsemibold', sans-serif;
+  text-decoration: none;
+  color: black;
+}
+
+button {
+  border: none;
+  background: none;
+  cursor: pointer;
+  outline: none;
+  display: flex;
+  align-items: center;
+  padding: 0 40px;
+  color: black;
+  font-size: 13px;
+  font-family: 'montserratbold', sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__pic {
+  margin: 10px;
+  width: 166px;
+  height: 51px;
+}
+
+.menu {
+  padding: 0;
+}
+
+.header__aside {
+  display: flex;
+  margin: 30px 0 0 0;
+}
+
+.header__menu-link {
+  display: inline-block;
+  margin: 0 20px;
+  color: black;
+  transition: color 0.1s liner;
+  position: relative;
+}
+
+.header__menu-link::after {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 2px;
+  background-color: #e51937;
+  opacity: 0;
+  position: absolute;
+  top: 102%;
+  left: 0;
+  z-index: 1;
+  transition: opacity 0.1s liner;
+}
+
+.header__menu-link:hover {
+  color: #e51937;
+}
+
+.header__menu-link:hover::after {
+  opacity: 1;
+}
+
+#light {
+  color: #e51937;
+  margin: 30px 50px 0 0;
+}
+
+.header__line {
+  position: absolute;
+  left: 458px;
+  top: 50px;
+}
+
+.menu__icon {
+  width: 24px;
+  height: 15px;
+  margin: 0 5px;
+}
+
+.main__title {
+  font-size: 90px;
+  font-family: 'montserratsemibold', sans-serif;
+  margin: 40px 0 0 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__subtitle {
+  font-size: 15px;
+  font-family: 'montserratmedium', sans-serif;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__pic {
+  width: 100%;
+  margin: -110px 0 0 0;
+}
+
+.main__info {
+  display: flex;
+  justify-content: center;
+}
+
+.info__content {
+  display: flex;
+}
+
+.main__content {
+  margin: 30px 0;
+}
+
+.main__content-item {
+  margin: 0 20px;
+  display: flex; 	
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)